fetch-todolist/src/shared/Input.js

```diff
@@ -0,0 +1,15 @@
+import { ErrorMessage, Field } from "formik";
+import TextError from "./TextError";
+
+import "../component/styling.css";
+const Input = (props) => {
```

Comment:
use prop types applicable for whole file

Reply:
ok